servicecontrol/plugins/index.md

```diff
@@ -32,15 +49,15 @@ NOTE: For NServiceBus version-dependent requirements for each plugin, review the
 
 Once deployed on an active endpoint, the endpoint sends plugin-specific information to ServiceControl. Plugins send messages using the defined endpoint transport to the ServiceControl queue. Location of ServiceControl queue is determined by the following:
 
-1. **Endpoint`s configuration file** 
+1. **Endpoint`s configuration file**
 Check for an `appSetting` named `ServiceControl/Queue` e.g. `<add key="ServiceControl/Queue" value="particular.servicecontrol"/>`.
-1. **Convention based on the configured Error Queue machine** 
+1. **Convention based on the configured Error Queue machine**
```

Comment:
@mikeminutillo I really don't know what to answer here, so pinging @indualagarsamy maybe she has an idea